src/bindings/python/tests/test_graph/test_convolution.py

```diff
@@ -5,143 +5,66 @@
 import numpy as np
 
 import openvino.runtime.opset8 as ov
-from tests.runtime import get_runtime
-from tests.test_graph.test_ops import convolution2d
-from tests.test_graph.util import run_op_node
 
 
 def test_convolution_2d():
 
     # input_x should have shape N(batch) x C x H x W
-    input_x = np.array(
-        [
-            [0.0, 0.0, 5.0, 5.0, 0.0, 0.0, 0.0, 0.0, 0.0],
-            [0.0, 0.0, 5.0, 5.0, 0.0, 0.0, 0.0, 0.0, 0.0],
-            [0.0, 0.0, 5.0, 5.0, 0.0, 0.0, 0.0, 0.0, 0.0],
-            [0.0, 0.0, 5.0, 5.0, 0.0, 0.0, 0.0, 0.0, 0.0],
-            [0.0, 0.0, 5.0, 5.0, 0.0, 0.0, 0.0, 0.0, 0.0],
-            [0.0, 0.0, 5.0, 5.0, 0.0, 0.0, 0.0, 0.0, 0.0],
-            [0.0, 0.0, 5.0, 5.0, 0.0, 0.0, 0.0, 0.0, 0.0],
-            [0.0, 0.0, 5.0, 5.0, 0.0, 0.0, 0.0, 0.0, 0.0],
-            [0.0, 0.0, 5.0, 5.0, 0.0, 0.0, 0.0, 0.0, 0.0],
-        ],
-        dtype=np.float32,
-    ).reshape(1, 1, 9, 9)
+    input_x = ov.parameter((1, 1, 9, 9), name="input_x", dtype=np.float32)
 
     # filter weights should have shape M x C x kH x kW
-    input_filter = np.array([[1.0, 0.0, -1.0], [2.0, 0.0, -2.0], [1.0, 0.0, -1.0]], dtype=np.float32).reshape(
-        1, 1, 3, 3,
-    )
+    input_filter = ov.parameter((1, 1, 3, 3), name="input_filter", dtype=np.float32)
 
     strides = np.array([1, 1])
     pads_begin = np.array([1, 1])
     pads_end = np.array([1, 1])
     dilations = np.array([1, 1])
+    expected_shape = [1, 1, 9, 9]
 
     # convolution with padding=1 should produce 9 x 9 output:
-    result = run_op_node([input_x, input_filter], ov.convolution, strides, pads_begin, pads_end, dilations)
-
-    assert np.allclose(
-        result,
-        np.array(
-            [
-                [
-                    [
-                        [0.0, -15.0, -15.0, 15.0, 15.0, 0.0, 0.0, 0.0, 0.0],
-                        [0.0, -20.0, -20.0, 20.0, 20.0, 0.0, 0.0, 0.0, 0.0],
-                        [0.0, -20.0, -20.0, 20.0, 20.0, 0.0, 0.0, 0.0, 0.0],
-                        [0.0, -20.0, -20.0, 20.0, 20.0, 0.0, 0.0, 0.0, 0.0],
-                        [0.0, -20.0, -20.0, 20.0, 20.0, 0.0, 0.0, 0.0, 0.0],
-                        [0.0, -20.0, -20.0, 20.0, 20.0, 0.0, 0.0, 0.0, 0.0],
-                        [0.0, -20.0, -20.0, 20.0, 20.0, 0.0, 0.0, 0.0, 0.0],
-                        [0.0, -20.0, -20.0, 20.0, 20.0, 0.0, 0.0, 0.0, 0.0],
-                        [0.0, -15.0, -15.0, 15.0, 15.0, 0.0, 0.0, 0.0, 0.0],
-                    ],
-                ],
-            ],
-            dtype=np.float32,
-        ),
-    )
+    node = ov.convolution(input_x, input_filter, strides, pads_begin, pads_end, dilations)
+    assert node.get_type_name() == "Convolution"
+    assert node.get_output_size() == 1
+    assert list(node.get_output_shape(0)) == expected_shape
```

Comment:
Testing of `node.get_output_element_type(0)` can be considered as additional assert.

Reply:
I'll add it wherever applicable, as Ilya mentioned - in the second part.